.travis.yml

```diff
@@ -12,22 +13,20 @@ cache:
 services:
   - memcached
   - redis-server
+  - mysql
 
 addons:
   postgresql: 10
   chrome: stable
   apt:
     sources:
-      - sourceline: "ppa:mc3man/trusty-media"
+      - sourceline: "ppa:jonathonf/ffmpeg-3"
       - sourceline: "ppa:ubuntuhandbook1/apps"
-      - mysql-5.7-trusty
     packages:
       - ffmpeg
       - mupdf
       - mupdf-tools
       - poppler-utils
-      - mysql-server
-      - mysql-client
       - postgresql-10
```

Comment:
Do we still need these lines?  Can not be specified in service?

Reply:
Removed packages from the global section and moved to the matrix where they are required.